velox/connectors/hive/HiveConfig.h

```diff
@@ -104,6 +104,9 @@ class HiveConfig {
   static constexpr const char* kOrcUseColumnNamesSession =
       "hive_orc_use_column_names";
 
+  static constexpr const char* kEnableParquetBloomFilter =
```

Comment:
what does this option do? Does it generate bloom filter when writing files, or uses them when reading files with bloom filter? Regardless, we should add comments and a PR description to clarify.